src/lib/hooks/useCurrency.ts

```diff
@@ -29,10 +103,7 @@ function parseStringOrBytes32(str: string | undefined, bytes32: string | undefin
  * Returns null if token is loading or null was passed.
  * Returns undefined if tokenAddress is invalid or token does not exist.
  */
-export function useTokenFromNetwork(
-  tokenAddress: string | null | undefined,
-  tokenChainId?: number
-): Token | null | undefined {
+export function useTokenFromNetwork(tokenAddress: string | undefined): Token | null | undefined {
```

Comment:
i don't think this is properly named now. this behavior is `useTokenFromCurrentWeb3ReactNetwork` or something like that right?